src/main/webapp/app/config/notification-middleware.ts

```diff
@@ -86,7 +86,7 @@ export default () => next => action => {
               break;
 
             case 404:
-              addErrorAlert('Not found', 'error.url.not.found');
+              addErrorAlert('Not found', 'error.urlNotFound');
```

Comment:
any specific reason you do not follow original style?

Reply:
Original style is not working as it return the translation is not found instead of the not found error.

Reply:
@LUXIANZE @junzai97 is this issue resolved?
If yes we can merge the PR if no issue